src/github.com/matrix-org/dendrite/common/httpapi.go

```diff
@@ -62,6 +87,12 @@ func MakeInternalAPI(metricsName string, f func(*http.Request) util.JSONResponse
 			span = tracer.StartSpan(metricsName, ext.RPCServerOption(clientContext))
 		}
 		defer span.Finish()
+
+		ext.HTTPUrl.Set(span, req.URL.String())
+		ext.HTTPMethod.Set(span, req.Method)
+
+		// TODO: Do we need to do the newStatusCodeResponseWriter stuff?
```

Comment:
Yes, seeing if an internal api succeeded or failed would be useful i think

Reply:
Ah, that's what that is for. Will do!